packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/controller.js

```diff
@@ -2,7 +2,15 @@ import { NO_OS_INSTALLED_REGEX, TRAVAUX_LINK } from './constants';
 
 export default class NutanixNodeGeneralInfoCtrl {
   /* @ngInject */
-  constructor($translate, coreConfig, NutanixNode, $rootScope, Alerter) {
+  constructor(
+    $state,
```

Comment:
**nitpick:** Dont inject `$state` into controller. Can you please get the `href` from resolve?

Reply:
sure